Sources_v3/Controllers/ChannelController/ChannelController.swift

```diff
@@ -667,16 +662,14 @@ public extension _ChatChannelController {
             return
         }
         
-        // Send stop typing event.
+        /// Send stop typing event.
```

Comment:
nit: Since this is just a normal comment, the original `//` was probably more correct than `///` ;-)